k8s/vizier/base/metadata_role.yaml

```diff
@@ -11,7 +11,6 @@ metadata:
 rules:
 - apiGroups:
   - ""
-  - "apps"
   resources:
```

Comment:
Can we remove `replicasets` and `deployments` from this group?

Reply:
Whoops! yes of course 😄 .  Fixed in latest.